.gitignore

```diff
@@ -1,5 +1,5 @@
 # NPM/Node
-/node_modules/
+node_modules/*
```

Comment:
Can we merge as is and track the [PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14850) with an issue?